src/modules/reservationPopup.js

```diff
@@ -0,0 +1,50 @@
+import closeSvg from '../assets/images/close.svg';
+
+const baseAPI = 'https://api.tvmaze.com/shows';
+
+const closePopup = () => {
+  const popup = document.querySelector('.popup');
+  popup.style.visibility = 'hidden';
+};
+
+const reservations = async (e) => {
+  const closeButton = document.getElementById('closeButton');
+  closeButton.src = closeSvg;
+  if (e && e.target && e.target.classList.contains('reservations')) {
+    const buttonId = parseInt(e.target.id, 10);
+    const headingSection = document.getElementById('headingSection');
+    const popupImg = document.getElementById('popup-img');
+    const projectDesc = document.getElementById('projectDesc');
+    const headValue = document.getElementById('headerSect');
+    const reserveBtn = document.querySelector('.form_btn');
+    const popup = document.querySelector('.popup');
+    const tempCard1 = document.querySelector('.start-date');
+    const tempCard2 = document.querySelector('.end-date');
+    const tempCard3 = document.querySelector('.movieComment');
+    const tempBtn = document.querySelector('.reserve_btn');
```

Comment:
- [ ] There is a little variance between your design and the wireframe, kindly adjust it. Ability to reproduce a given template shows how well one pay attention to details. 👍 

## Wireframe
<img width="146" alt="Untitled" src="https://github.com/Kanyorok/Capstone-Grp_Kanban/assets/103386183/4d6acc18-f262-444b-88c6-eee79d9b6106">

## Your Design
 
<img width="947" alt="Untitled" src="https://github.com/Kanyorok/Capstone-Grp_Kanban/assets/103386183/aa4cec06-9080-4bba-9c23-fb640a843736">

Reply:
@Peter-Kibet - you didn't implement this requested change from the previous CR - as stated in the [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md#:~:text=You%20should%20follow%20the%20layout%20of%20the%20wireframes%20provided).